src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -57,6 +57,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     mentors.add(Richi);
 
     String jsonMentors = new Gson().toJson(mentors);
+    System.out.println("HELLLLLLLOOOOO" + jsonMentors);
```

Comment:
Is this intended to be put into master? Lol

Reply:
Omg no! Just a debugging comment xo